include/cantera/oneD/StFlow.h

```diff
@@ -19,13 +19,16 @@ namespace Cantera
 //   constants
 //------------------------------------------
 
-// Offsets of solution components in the solution array.
-const size_t c_offset_U = 0; // axial velocity
-const size_t c_offset_V = 1; // strain rate
-const size_t c_offset_T = 2; // temperature
-const size_t c_offset_L = 3; // (1/r)dP/dr
-const size_t c_offset_E = 4; // electric poisson's equation
-const size_t c_offset_Y = 5; // mass fractions
+//! Offsets of solution components in the 1D solution array.
+enum offset1D
+{
+    c_offset1D_U   //! axial velocity
+    , c_offset1D_V //! strain rate
+    , c_offset1D_T //! temperature
+    , c_offset1D_L //! (1/r)dP/dr
+    , c_offset1D_E //! electric poisson's equation
+    , c_offset1D_Y //! mass fractions
```

Comment:
Thanks for fixing the comments (the table looks great).

However, I don't think that the additional `1D` adds any useful information to this class, as it is part of `Sim1D` inside the `oneD` folder. I'd prefer if you could revert to the old nomenclature. Thanks!